index.html

```diff
@@ -20,7 +20,8 @@
 			<p>And here's a snippet you can use to put this in your Markdown documents:</p>
 			<p><code>[![GPL enforced badge](https://img.shields.io/badge/GPL-enforced-blue.svg "This project enforces the GPL.")](https://gplenforced.org)</code></p>
 			<p>You'll note that the badge links to this page and shows some text when you hover over it. The badge itself is SVG (i.e. a vector graphic) thanks to <a href="https://shields.io">shields.io</a>.</p>
-			<p>Want to do more? Donating to the <a href="https://sfconservancy.org/">Software Freedom Conservancy</a> is a great way to contribute to community GPL enforcement.</p>
+			<p>Want to do more? Donating to or helping the GPL compliance efforts of the <a href="https://www.fsf.org/licensing/compliance">Free Software Foundation</a>, <a href="https://sfconservancy.org/copyleft-compliance/">Software Freedom Conservancy</a> or <a href="http://gpl-violations.org/helping/">gpl-violations.org</a> is a great way to contribute to community GPL enforcement.</p>
+                        <p>More information about complying with the GPL is available at <a href="https://copyleft.org/">copyleft.org</a>.</p>
```

Comment:
This indentation is off.

Also, I think this would be better suited to the "what is enforcement?" section above.

Thoughts?